NOTICES.md

```diff
@@ -50,6 +51,22 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 
+>>>> github.com/pyca/cryptography - 3.3.2
```

Comment:
please move this to be the first location

Reply:
Done